zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/remote/RemoteInterpreterServer.java

```diff
@@ -178,6 +187,8 @@ public RemoteInterpreterServer(String intpEventServerHost,
     server = new TThreadPoolServer(
         new TThreadPoolServer.Args(serverTransport).processor(processor));
     remoteWorksResponsePool = Collections.synchronizedMap(new HashMap<String, Object>());
+
+    clusterManagerClient.start(interpreterGroupId);
```

Comment:
Even in single node deployment, this code is still called ?

Reply:
All interpreter processes report their status to the cluster.
After the zeppelin cluster has metadata for all zeppelin services and interpreter processes,
It will be of great help to the management of zeppelin in the future.
E.g,
1. Cluster metadata can know which interpreter processes are started on which servers.
2. The operating state of each interpreter process,
3. CPU and memory resource usage for each interpreter process.
4. Even the user's session information can be managed uniformly.

Therefore, I consider that the single machine is also managed by cluster metadata.